src/Jira/Api.php

```diff
@@ -551,20 +551,27 @@ public function createIssue($project_key, $summary, $issue_type, array $options
 	 * @param integer $start_at    Start at.
 	 * @param integer $max_results Max results.
 	 * @param string  $fields      Fields.
+     * @param array   $expanded    Fieldsets to expand in returned data
 	 *
 	 * @return Result|false
 	 */
-	public function search($jql, $start_at = 0, $max_results = 20, $fields = '*navigable')
+	public function search($jql, $start_at = 0, $max_results = 20, $fields = '*navigable', $expanded = array())
 	{
+	    $data = array(
+            'jql' => $jql,
+            'startAt' => $start_at,
+            'maxResults' => $max_results,
+            'fields' => $fields
+        );
+
+	    if (count($expanded) > 0) {
```

Comment:
CS formatting + wrong indentation. Please look at other code and validate your changes with `phpcs` as written on https://github.com/chobie/jira-api-restclient/blob/master/CONTRIBUTING.md page.